mod-source-record-storage-server/src/main/java/org/folio/dao/RecordDaoImpl.java

```diff
@@ -1286,9 +1307,10 @@ private MarcBibCollection toMarcBibCollection(QueryResult result) {
   }
 
   @Override
-  public Future<Record> saveUpdatedRecord(ReactiveClassicGenericQueryExecutor txQE, Record newRecord, Record oldRecord) {
+  public Future<Record> saveUpdatedRecord(ReactiveClassicGenericQueryExecutor txQE, Record newRecord, Record oldRecord, Map<String, String> okapiHeaders) {
     LOG.trace("saveUpdatedRecord:: Saving updated record {}", newRecord.getId());
-    return insertOrUpdateRecord(txQE, oldRecord).compose(r -> insertOrUpdateRecord(txQE, newRecord));
+    return insertOrUpdateRecord(txQE, oldRecord).compose(r -> insertOrUpdateRecord(txQE, newRecord))
```

Comment:
Here we are invoking "insertOrUpdateRecord" for both oldRecord and newRecord. So, I think we should emit the event for both oldRecord and newRecord.

Reply:
I don't think we should do this: only SRS stores multiple versions of the same entity, right?
Other systems should get update message only, otherwise there will be just 2 update messages with the same recordId one after another